recipes/kalasiris/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "kalasiris" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bdcb44b71dd7c3a0cf3deb42d18a7516098cb2a372710433413c0d1ac9449db
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

It looks like this can be a noarch package:
```suggestion
  noarch: python
```

---

Ah, now I can have the noarch, this makes sense.